src/cbsodatav4_kcvanderlinden/cbsodatav4.py

```diff
@@ -2,142 +2,188 @@
 import requests
 from requests.adapters import HTTPAdapter
 from urllib3.util import Retry
-import concurrent.futures
 import os
-import math
+import sys
+import concurrent.futures
+import json
+import dask.dataframe as dd
+import dask
+import random
+from time import sleep
 
 basic_odata_url = 'https://datasets.cbs.nl/odata/v1/CBS/'
 
+def DataFrame(tableID:str, name:str=None, limit:int=None, dataFilter:str=None, save_csv_path:str=None):
+    """
+    Return a Pandas DataFrame containing the data from the CBS OData API.
+
+    Parameters
+    ----------
+    tableID : str
+        The ID of the table to retrieve data from.
+    name : str, optional
+        The name of the specific table to retrieve, by default None.
+        If not specified, the entire dataset will be retrieved.
+    limit : int, optional
```

Comment:
ik zie in meedere functie limit terugkomen, mogelijk kun je deze specificeren, welke limit het waar om gaat.